fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/FluentGlobalTokens.kt

```diff
@@ -218,6 +218,16 @@ object FluentGlobalTokens {
         ReplaceWith("token.value"))
     fun elevation(token: ShadowTokens): Dp = token.value
 
+    enum class ElevationTokens(val value: Dp) {
+        Elevation00(0.dp),
```

Comment:
let's use the existing ShadowTOkens

Reply:
Updated

Reply:
I can't see the change. has it been pushed?

Reply:
Yes, pushed now